crates/uv/src/commands/project/mod.rs

```diff
@@ -1730,7 +1730,7 @@ pub(crate) async fn resolve_names(
 }
 
 #[derive(Debug, Clone)]
-pub(crate) enum PreferenceSource<'lock> {
+pub(crate) enum PreferenceLocation<'lock> {
```

Comment:
This renamed to disambiguate from `Source`

Reply:
Yeah that's good.